Modules/FindCUDA/select_compute_arch.cmake

```diff
@@ -0,0 +1,175 @@
+# BorIFrowed from caffe
```

Comment:
The main documentation needs to be in FindCUDA where the document generation will parse it out.